requirements_dev.txt

```diff
@@ -1,2 +1,3 @@
 pytest
 perf >= 0.8.0
+wcwidth
```

Comment:
It is listed as direct dependency in `setup.py`, are you sure you want to add it to _dev?